src/components/ConfirmPopup.js

```diff
@@ -0,0 +1,45 @@
+import Popup from "./Popup";
+
```

Comment:
* Please, make the `confirmation` modal matc the common `mobile` design https://snipboard.io/Ib69uM.jpg